samples/java-eventsourced-shopping-cart/src/main/proto/com/example/shoppingcart/domain/shoppingcart_domain.proto

```diff
@@ -5,20 +5,7 @@
 syntax = "proto3";
 
 package com.example.shoppingcart.domain; // <1>
-
-import "akkaserverless/annotations.proto"; // <2>
-
-option java_outer_classname = "ShoppingCartDomain"; // <3>
-
-// tag::event_sourced_entity[]
-// Describes how this domain relates to an event sourced entity
-option (akkaserverless.file).event_sourced_entity = { // <4>
-  name: "ShoppingCart" // <5>
-  entity_type: "shopping-cart" // <6>
-  state: "Cart" // <7>
-  events: ["ItemAdded", "ItemRemoved"] // <8>
-};
-// end::event_sourced_entity[]
+option java_outer_classname = "ShoppingCartDomain"; // <2>
```

Comment:
oh, I still need to clean up this. That will make the docs to break. 

Actually I naïvely tried to fixed this today by swapping the descriptor-sets folders in d17d48cb8028c205db0c321d3feca055021b7534. Silly, the problem in the docs are the files, not the descriptors. 🤦